### ISSUE TITLE
User file with large ZoneHVAC:CoolingPanel:RadiantConvective:Water crashes in UA calculation

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6098]
Issue overview
--------------
User file crashes with log of a negative number [here]() when calculating the UA for the radiant cooling panel.
```
			CoolingPanel( CoolingPanelNum ).UA = -MDotXCp * log( 1.0 - ( std::abs(Qrated) / ( MDotXCp * std::abs( Tinletr - Tzoner ) ) ) );
```
Qrated is 6498W and the denominator is only 3115.  This results in log(1.0 - 2.2) which fails.

The calculation needs to be protected from a bad log argument in general.
But this raises questions about the usefulness of the three Rated inputs in this object:
```
    16,                      !- Rated Inlet Water Temperature {C}
    28,                      !- Rated Inlet Space Temperature {C}
    0.063,                   !- Rated Water Mass Flow Rate {kg/s}
```
There is no corresponding input for a Rated Capacity, so the large actual capacity is compared to these Rated inputs to calculate the UA value.  Seems that the mass flow rate corresponding to the "Maximum Chilled Water Flow Rate" should be used here instead (which is more like 0.5).  

And the Rated inputs could possibly be unnecessary?

### Workaround
Enter a larger value for Rated Water Mass Flow Rate, check the eio output for the autosized Max Chilled Water Flow Rate and then use the mass flow equivalent as the input for Rate Water Mass Flow Rate.

### Details
Some additional details for this issue (if relevant):
 - Win10, 64-bit
 - Version of EnergyPlus 8.7.0
 - Helpdesk ticket number 11974

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added to EnergyPlusDevSupport\DefectFiles
 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by RKStrand** [comment]
@mjwitte I'll start looking at this, hope to get to this over the next several days.  I have added a ticket to Pivotal (ID #147520311).  From an initial look at this, it seems that the equation used for the UA calculation assumes that the scaled capacity is the rated capacity which is not correct. I suggest that a check be put into the UA calculation to avoid an improper log calculation but will think more about the inputs and how to resolve the scaled vs. rated issue.

**Comment by RKStrand** [comment]
@mjwitte Ok, this is what I propose to resolve this issue:

1. Institute a check on the calculation and throw appropriate error messages when the log part of the calculation would cause a crash.
2. Modify the documentation in the I/O Ref to clearly define the connection between the rated temps and flow rate and the capacity of the unit (can't have a more than 100% efficient device).
3. The usual test suite thing.

What do you think?

**Comment by RKStrand** [comment]
@mjwitte Implemented the above...pull request made...ready for review.

**Comment by RKStrand** [comment]
@mjwitte Any idea why this one is still showing up as not fully closed off?

**Comment by mjwitte** [comment]
Probably just an oversight at merge time.  Closed via #6181.
